### ISSUE TITLE
Request: more comprehensive sample, incl. component usage examples

### ISSUE THREAD

**Issue by mayel** [Microsoft/reactxp#47]
Looks like the docs and sample assume working knowledge of React/RN, but would be great to expound more for developers who are starting out with ReactXP with no prior experience (other than JS and general web development know-how).

**Comment by erictraut** [comment]
@mayel, thanks for your suggestion. I extended the "Hello World" sample with a toggle switch component that demonstrates some of the capabilities of ReactXP, including styling, animations, and event handling. 

As for learning React concepts, I've provided a high-level overview in the ReactXP documentation:
[React Concepts](https://microsoft.github.io/reactxp/docs/react_concepts.html)
[Component Lifecycle](https://microsoft.github.io/reactxp/docs/react_lifecycle.html)
etc.

For more detailed documentation about React including tutorials and samples, check out Facebook's [React](React
) and [React Native](http://facebook.github.io/react-native/) sites.
